tests/core/test_elf_file.cpp

```diff
@@ -51,7 +51,7 @@ namespace LinuxDeployTest {
     }
 
     TEST_F(ElfFileTest, checkInvalidElfHeaderOnEmptyFile) {
-        expectThrowsElfFileErrorInvalidElfHeader("/dev/null");
+        expectThrowsElfFileErrorInvalidElfHeader(EMPTY_FILE_PATH);
```

Comment:
Good idea!

Reply:
The change was needed since `is_regular_file` return `false` for `/dev/null`